### PR TITLE
Write about the beta of version 2.10

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 39cdb95b755117ca5b21b45f12f16f4bcf348561

https://deploy-preview-229--openbuildservice.netlify.com

File: _posts/releases/2019-06-17-version-2.10-beta.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  below /message/ are deprecated and will be removed in the next
+  version.
+* Deprecated Ratings. The following API routes are deprecated and will be removed
+  in the next version:
+  - GET /statistics/highest_rated?limit=<limit>
+  - GET /statistics/rating/<project>/<package>
+  - PUT /statistics/rating/<project>/<package>
+* Switch most frontend services from sysv init scripts to systemd units (obs-api-support.target)
+
+## More Information
```
Comment (hennevogel) [inline _posts/releases/2019-06-17-version-2.10-beta.md]: Can't those all be 'partials'?

Comment (dmarcoux) [reply _posts/releases/2019-06-17-version-2.10-beta.md]: Good idea, I'll do it.

Comment (dmarcoux) [reply _posts/releases/2019-06-17-version-2.10-beta.md]: Done

File: _posts/releases/2019-06-17-version-2.10-beta.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+This version is the first one with the revamped user interface.
+For details on this, please refer to our blog where we have written a
+[series of post](https://openbuildservice.org/2018/10/05/revamping-ui/) on the matter.
+
+TODO: Other highlights?
+
+## Known Issues
+
+TODO: Something here?
```
Comment (dmarcoux) [inline _posts/releases/2019-06-17-version-2.10-beta.md]: Should I remove this section completely?

File: _posts/releases/2019-06-17-version-2.10-beta.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+category: releases
+---
+
+We have released the beta of the upcoming Open Build Service (OBS) version 2.10.
+
+This version is the first one with the revamped user interface.
+For details on this, please refer to our blog where we have written a
+[series of post](https://openbuildservice.org/2018/10/05/revamping-ui/) on the matter.
+
+TODO: Other highlights?
```
Comment (dmarcoux) [inline _posts/releases/2019-06-17-version-2.10-beta.md]: Would it be enough if it stays as it is? So no more highlights.

**Comment by dmarcoux** [comment]
@hennevogel Please have another look.

File: _posts/releases/2019-06-17-version-2.10-beta.md (hunk 2 above)
Comment (hennevogel) [reply _posts/releases/2019-06-17-version-2.10-beta.md]: you could write something like: None yet...

File: _posts/releases/2019-06-17-version-2.10-beta.md (hunk 3 above)
Comment (hennevogel) [reply _posts/releases/2019-06-17-version-2.10-beta.md]: Yeah I think that would be okay. You could rephrase it like: Look ma' this is the first bootstrap release, pay attention!

**Comment by dmarcoux** [comment]
[Beta Release Blog Post](https://trello.com/c/lEtYnqG4/1172-beta-release-blog-post)

File: _posts/releases/2019-06-17-version-2.10-beta.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+For details on this, please refer to our blog where we have written a
+[series of post](https://openbuildservice.org/2018/10/05/revamping-ui/) on the matter.
+
+## Known Issues
+
+None.
+
+## New Features
+
+* Add binary release tracking data for containers.
```
Comment (DavidKang) [inline _posts/releases/2019-06-17-version-2.10-beta.md]: We have a mix of items with final dot and other without it.

**Pull request merged**